grpc/src/lib.rs

```diff
@@ -55,6 +58,33 @@ impl GrpcOctree {
             root_bounding_cube,
         }
     }
+
+    pub fn get_points_in_box(&self, bounding_box: &Aabb3<f32>) -> Vec<Point3<f32>> {
+        let mut req = proto::GetPointsInBoxRequest::new();
+        req.mut_bounding_box().mut_min().set_x(bounding_box.min.x);
+        req.mut_bounding_box().mut_min().set_y(bounding_box.min.y);
+        req.mut_bounding_box().mut_min().set_z(bounding_box.min.z);
+        req.mut_bounding_box().mut_max().set_x(bounding_box.max.x);
+        req.mut_bounding_box().mut_max().set_y(bounding_box.max.y);
+        req.mut_bounding_box().mut_max().set_z(bounding_box.max.z);
+        let mut replies = self.client.get_points_in_box(&req).unwrap();
+        let mut points = Vec::new();
+        loop {
+            let f = replies.into_future();
```

Comment:
use replies.for_each() which returns a future on which you can then wait. No need for the loop.